src/components/markdown/BlogPostPage.tsx

```diff
@@ -54,8 +54,11 @@ export const BlogPostPage: React.FunctionComponent<{
 
       <h1 className="pageTitle">{page.title}</h1>
 
-      <div className="postAuthor">{page.author}</div>
-      <div className="postDate">{page.date}</div>
+      <div className="postMeta">
```

Comment:
Could we pick a different name for this class? Meta is confusing with metadata, which we also have for every post (title, social image, etc).

Reply:
Will do.

Reply:
Done